RELEASE_NOTES.md

```diff
@@ -1,12 +1,12 @@
 Release Notes
 =============
 
-v1.0.0-alpha.x
+v1.0.0-alpha.15
 ---------------
 
 ### Breaking changes
 
-- Minimum OpenAssetIO version increased to v1.0.0-beta.3.0 to make use
+- Minimum OpenAssetIO version increased to v1.0.0-beta.2.1 to make use
```

Comment:
[Question] This isn't actually declared anywhere in the repo is it apart from here? It's just implicit?

Reply:
Yeah it is/was specified in the `pyproject.toml`.  Looks like we may have missed a couple of steps in e741fb51, i.e. not fully reverted 28792152

Reply:
Ah, I misread the changelog, thought that had been _added_ for the retargeting, not removed. Added a commit before the release one to add the dependency. https://github.com/OpenAssetIO/OpenAssetIO-Manager-BAL/pull/108/commits/07f5522598abfc0ded57658b9ee6877eaaf0bebf

Reply:
What about the other changes from https://github.com/OpenAssetIO/OpenAssetIO-Manager-BAL/commit/28792152a4550edc647fcadf7ae02a3a0c33c6ec that were not fully reverted?  We could continue to target main for the other CI jobs, but its a bit inconsistent.

Reply:
Did I make a total hash of it? Wasn't intentional ... should have done a legit revert instead of a bodge job. Will fix it up.

Reply:
Another commit to finish it https://github.com/OpenAssetIO/OpenAssetIO-Manager-BAL/pull/108/commits/860d78b41bf929669087ba5cda67408232f03ddc